ecosystem/sep-0008.md

```diff
@@ -88,79 +83,62 @@ Parameters:
 Name | Data Type | Description
 -----|-----------|------------
 status|string|`"success"`
-tx|string|Transaction Envelope XDR, Base64 encoded. This transaction will have both the original signature(s) from the request, as well as an additional issuer signature.
+tx|string|Transaction envelope XDR, base64 encoded. This transaction will have the original signature(s) from the request as well as the asset issuer's signature.
```

Comment:
💡 If we are replacing this SEP I think we should rename `tx` to `transaction` because we don't use the tx abbreviation in other SEPs that surface APIs and consistency in naming can help reduce cognitive load and downstream consistency in SDKs and code. e.g. SEP-10, SEP-30, SEP-24.

Reply:
https://github.com/stellar/stellar-protocol/pull/750/commits/2254a830f9bb5fe107e4c166c21cb1f5d46ca42f